packages/react-ui/src/field/Select.js

```diff
@@ -2,44 +2,64 @@ import React from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 
-import { animations, box, fonts, spacings } from "../theme";
+import { ArrowDown } from "../icons";
+import { animations, box, colors, fonts, spacings } from "../theme";
 
-export const Select = ({ children, ...props }) => (
-  <StyledSelect {...props}>{children}</StyledSelect>
+export const Select = ({ children, disabled, ...props }) => (
+  <StyledWrapper>
+    <StyledSelect disabled={disabled} {...props}>
+      {children}
+    </StyledSelect>
+    <StyledArrowDown isdisabled={disabled}>
```

Comment:
we should add aria-hidden + pointer-event: none

Reply:
You right